EFCountingLabel/Classes/EFCount.swift

```diff
@@ -47,7 +47,42 @@ extension EFCount {
 }
 
 public class EFCounter {
+
+    public enum RefreshRate: Int {
```

Comment:
`@objc public enum RefreshRate: Int` would allow any Int value instead of some predefined ones.
Or better, don't declare any enum and use:
```
/// Refresh rate in frames per second.
public var refreshRate: Int
```

Reply:
I think better to limit set of possible values. For example frame rate `25` will not work and will be changed by system to `30`. Guess it will be better to change `RefreshRate` to `frameInterval` (i.e. similar to deprecated `CADisplayLink.frameInterval`)